app/src/main/java/org/simple/clinic/editpatient/PatientEditScreen.kt

```diff
@@ -193,6 +196,34 @@ class PatientEditScreen(context: Context, attributeSet: AttributeSet) : Relative
     }
   }
 
+  fun hideValidationErrors(errors: Set<PatientEditValidationError>) {
+    errors.forEach {
+      when (it) {
+        FULL_NAME_EMPTY -> {
+          showEmptyFullNameError(false)
+        }
+
+        PHONE_NUMBER_EMPTY,
+        PHONE_NUMBER_LENGTH_TOO_SHORT,
+        PHONE_NUMBER_LENGTH_TOO_LONG -> {
+          showLengthTooShortPhoneNumberError(false)
```

Comment:
All these errors map to the same action, i.e, clearing the error on the same field.

Reply:
Do you feel it's better to add another method to just clear the error on the phone number field?

Reply:
I don't actually want to do this because then testing the validation errors as functions is hard. With the current setup, validating errors is extremely easy.

Reply:
I want to avoid behaviour testing in favour of value testing, that's why I set up the current methods the way they are right now.

Reply:
Sure okay